### PR TITLE
Minor formatting changes.

### PR DESCRIPTION
**Pull request by arfon**
For JOSS submission...

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -63,11 +63,11 @@ Platforms such as MassBank [@horai_massbank_2010], GNPS [@Wang2016] or MetaboLig
 Without suitable tools, however, exploitation of such datasets remains overly challenging. 
 In particular, large collected datasets contain data acquired using different instruments and measurement conditions, and can further contain a significant fraction of inconsistent, wrongly labeled, or incorrect metadata (annotations).
 
-``matchms`` is an open-access Python package to import, process, clean, and compare mass spectrometry data (MS/MS) (see \autoref{fig:flowchart}).
+``matchms`` is an open-source Python package to import, process, clean, and compare mass spectrometry data (MS/MS) (see \autoref{fig:flowchart}).
```
Comment (arfon) [inline paper/paper.md:66]: I think you mean open source here (not open access).

File: paper/paper.md [hunk 2]
```diff
@@ -63,11 +63,11 @@ Platforms such as MassBank [@horai_massbank_2010], GNPS [@Wang2016] or MetaboLig
...
-Raw data can be imported from the commonly used formats msp, mzML [@martens_mzmlcommunity_2011], mzXML, MGF (mzML, mzXML, MGF file importers are built on top of pyteomics [@levitsky_pyteomics_2019][@goloborodko_pyteomicspython_2013]), as well as from json files (as provided by GNPS), but also via Universal Spectrum Identifiers (USI) [@wang_interactive_2020]. Further data formats or more extensive options regarding metadata parsing can best be handled by using pyteomics [@levitsky_pyteomics_2019] or pymzml [@kosters_pymzml_2018].
+Raw data can be imported from the commonly used formats msp, mzML [@martens_mzmlcommunity_2011], mzXML, MGF (mzML, mzXML, MGF file importers are built on top of pyteomics [@levitsky_pyteomics_2019;@goloborodko_pyteomicspython_2013], as well as from JSON files (as provided by GNPS), but also via Universal Spectrum Identifiers (USI) [@wang_interactive_2020]. Further data formats or more extensive options regarding metadata parsing can best be handled by using pyteomics [@levitsky_pyteomics_2019] or pymzml [@kosters_pymzml_2018].
```
Comment (arfon) [inline paper/paper.md:68]: This is a minor formatting fix for the citation rendering.

File: paper/paper.md [hunk 3]
```diff
@@ -63,11 +63,11 @@ Platforms such as MassBank [@horai_massbank_2010], GNPS [@Wang2016] or MetaboLig
...
 ``matchms`` contains numerous metadata cleaning and harmonizing filter functions that can easily be stacked to construct a desired pipeline (\autoref{fig:filtering}), which can also easily be extended by custom functions wherever needed. Available filters include extensive cleaning, correcting, checking of key metadata fields such as compound name, structure annotations (InChI, SMILES, InChIKey), ionmode, adduct, or charge.
-Many of the provided metadata cleaning filters were designed for handling and improving GNPS-style MGF or json datasets. For future versions, however, we aim to further extend this to other commonly used public databases.
+Many of the provided metadata cleaning filters were designed for handling and improving GNPS-style MGF or JSON datasets. For future versions, however, we aim to further extend this to other commonly used public databases.
```
Comment (arfon) [inline paper/paper.md:70]: JSON is an acronym.

File: paper/paper.md (hunk 1 above)
Comment (florian-huber) [reply paper/paper.md:66]: Yes, indeed. Thanks!

File: paper/paper.md (hunk 2 above)
Comment (florian-huber) [reply paper/paper.md:68]: Perfect, thanks.

File: paper/paper.md (hunk 3 above)
Comment (florian-huber) [reply paper/paper.md:70]: True, thanks!

**Pull request merged**